forge-gui/res/languages/es-ES.properties

```diff
@@ -2963,6 +2966,6 @@ lblBoon=Boon
 lblExitToWoldMap=Salir al mapa del mundo?
 lblStartArena=¿Quieres ir a la arena?
 lblWouldYouLikeDestroy=¿Le gustaría destruir {0}?
-lblAdventureDescription=�El modo de aventura es donde exploras el paisaje siempre cambiante de Shandalar, y criaturas de duelo para ganar oro, fragmentos, equipos y nuevas tarjetas para convertirte en el mejor y recogerlas a todos!
-lblEmptyDeck=Baraja de cartas vac�a
+lblAdventureDescription=�El modo de aventura es donde exploras el paisaje siempre cambiante de Shandalar, y criaturas de duelo para ganar oro, fragmentos, equipos y nuevas tarjetas para convertirte en el mejor y recogerlas a todos!
+lblEmptyDeck=Baraja de cartas vac�a
```

Comment:
formatting broke?

Reply:
Ugh 👀 taking a look

Reply:
looks like this in Diff to me, but maybe it's my browser?
![grafik](https://user-images.githubusercontent.com/8506892/232884946-015cda9f-6fea-46e8-87f6-c1abec76e35e.png)

Reply:
No something corrupted it 😞